### PR TITLE
chore: lint e2e

### PR DESCRIPTION
**Pull request by faddat**
## Description

Lint e2e

### Commit Message / Changelog Entry

```bash
chore: lint e2e
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by DimitrisJim — APPROVED** [review]
changes lgtm

File: e2e/tests/upgrades/upgrade_test.go [hunk 1]
```diff
@@ -680,18 +679,6 @@ func (s *UpgradeTestSuite) RegisterInterchainAccount(ctx context.Context, chain
 	s.AssertTxSuccess(txResp)
 }
 
-// getICAVersion returns the version which should be used in the MsgRegisterAccount broadcast from the
-// controller chain.
-func getICAVersion(chainAVersion, chainBVersion string) string {
```
Comment (damiannolan) [inline e2e/tests/upgrades/upgrade_test.go:685]: this is now unused?

**Review by damiannolan — APPROVED** [review]
Thanks @faddat, appreciate the clean up! 

Personally not a big fan of the `t.Helper()` stuff as I find it just kind of adds visual noise (for me) but I'll trust that its useful to someone!

File: e2e/relayer/relayer.go [hunk 2]
```diff
@@ -67,18 +70,18 @@ func newHermesRelayer(t *testing.T, tag string, logger *zap.Logger, dockerClient
 }
 
 // RelayerMap is a mapping from test names to a relayer set for that test.
-type RelayerMap map[string]map[ibc.Wallet]bool
```
Comment (DimitrisJim) [inline e2e/relayer/relayer.go:70]: > type name will be used as `relayer.RelayerMap` by other packages, and that stutters; consider calling this Map

I am a-a-amazed

File: e2e/tests/upgrades/upgrade_test.go (hunk 1 above)
Comment (DimitrisJim) [reply e2e/tests/upgrades/upgrade_test.go:685]: Yup! Removed here https://github.com/cosmos/ibc-go/pull/3667#discussion_r1206468677

**Comment by faddat** [comment]
@damiannolan how do you code?

I'm pretty VSCode or Cursor focused in terms of tools, tho I jump into neovim from time to time.  I think this matters for the t.Helper() stuff, because when in VSCode, t.Helper() calls do pretty amazing stuff.  You get in-line traces with a lot more detail than without it.

**Pull request merged**